data/transactions/logic/opcodes.go

```diff
@@ -598,11 +657,52 @@ var OpSpecs = []OpSpec{
 	{0xd1, "block", opBlock, proto("i:a"), randomnessVersion, field("f", &BlockFields)},
 }
 
+// OpNames lists all spec names in order they are introduced in the OpSpecs table
+var OpNames = make([]string, 0)
+
 type sortByOpcode []OpSpec
 
-func (a sortByOpcode) Len() int           { return len(a) }
-func (a sortByOpcode) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
-func (a sortByOpcode) Less(i, j int) bool { return a[i].Opcode < a[j].Opcode }
+func (a sortByOpcode) Len() int      { return len(a) }
+func (a sortByOpcode) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+func (a sortByOpcode) Less(i, j int) bool {
+	first := GetFullCode(a[i])
+	second := GetFullCode(a[j])
+	for k := range first {
+		if len(second) == k {
+			return false
+		}
+		if first[k] != second[k] {
+			return first[k] < second[k]
+		}
+	}
+	return true
+}
+
+// Serialize is used to write multi-byte opcodes as strings
+func Serialize(bytes []byte) string {
```

Comment:
This doesn't seem like something we should be exporting.

Reply:
Moved into opdoc for now, but I think later on we may move it back and export it for some better erroring on multi-ops